### ISSUE TITLE
Why is Windows Security preventing Moosync from opening...

### ISSUE THREAD

**Issue by Ares-03** [Moosync/Moosync#333]
Describe the bug
I just tried to install Moosync Portable and Moosync but after clicking to open it windows security showed this problem. My question is why is this problem appearing and is Moosync unsafe..

Screenshots 
![image](https://github.com/Moosync/Moosync/assets/106290878/dc30cefb-8cfa-4ffb-b35b-652aacc9da4c)

- OS: Windows
- Version - 10

**Comment by Ovenoboyo** [comment]
Yeah to have this screen not show up, I'd need to buy a signing certificate which is like $200+ and I definitely cannot afford that. Rest assured, moosync is not unsafe

**Comment by Ares-03** [comment]
> Yeah to have this screen not show up, I'd need to buy a signing certificate which is like $200+ and I definitely cannot afford that. Rest assured, moosync is not unsafe

Oh okay, and thanks for the reply..
